docs/samples/customdomain/README.md

```diff
@@ -0,0 +1,134 @@
+# Setting up custom domain name
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. You have a custom domain configured to route incoming traffic to the cluster's Ingress gateway.
+
+## Create the Ingress resource
+
+Edit the `knative-ingress.yaml` file to add your custom wildcard domain to the `spec.rules.host` section, replacing `<*.custom_domain>` with your custom wildcard domain. This is so that all incoming network traffic from your custom domain and any subdomain is routed to the `kfserving-ingressgateway`.
+
+```
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: knative-ingress
+  namespace: istio-system
+spec:
+  rules:
+    - host: "<*.custom_domain>"
+      http:
+        paths:
+          - backend:
+              serviceName: kfserving-ingressgateway
+              servicePort: 80
+            path: /
+```
+
+Apply the Ingress resource
+
+```
+kubectl apply -f knative-ingress.yaml
```

Comment:
maybe call it kfserving-ingress.yaml since we are using it in the context of KFServing.

Reply:
I am a bit confused here, since we are using istio as ingress gateway why do we need anther Ingress here to direct to istio ingress gateway?

Reply:
+1. One ingress please!

Reply:
Many cloud platforms provide default domains which are already routed to a Kubernetes Ingress. The additional Ingress is required to route the traffic of the default domain to the istio ingress gateway. This can be skipped if the custom domain is configured to the istio-ingressgateway/kfserving-ingressgateway's IP address/Load Balancer. I'll make this edit in the README